src/traversals/bfs.jl

```diff
@@ -189,12 +189,15 @@ function is_bipartite(g::SimpleGraph, s::Int)
     return visitor.is_bipartite
 end
 
-function is_bipartite(g::SimpleGraph)
-    nvg = nv(g)
-    for v in vertices(g)
-        if !is_bipartite(g, v)
-            return false
+if VERSION >= v"0.4.0-dev+6371"
+    is_bipartite(g::SimpleGraph) = all([is_bipartite(g,v) for v in vertices(g)])
```

Comment:
That looks fine. But is there a performance penalty to using a lambda like this? (I don't quite get the recent discussions about functors/lambdas... or if that's even relevant.) If there is a performance penalty then I'm not sure this change is worth it!